wl-clipboard-rs-tools/Cargo.toml

```diff
@@ -12,7 +12,7 @@ keywords.workspace = true
 categories = ["command-line-utilities"]
 
 [dependencies]
-anyhow = "1.0.96"
+anyhow = "1.0.97"
```

Comment:
Why would you do this? Does wl-clipboard-rs somehow hard-require some changes made in a patch release? Nor am I seeing this PR changing anything in the code that would introduce such a requirement?

Reply:
When I test the project before releasing, cargo will resolve the latest dependency versions. Therefore, I make my Cargo.toml reflect the versions that I tested against.

Reply:
I primarily test right before tagging a new release. I also bump all dependencies before tagging a new release and testing it. However, it's easier to catch regressions when bumping versions one by one like this as they come out rather than all at once before release, because I am running the git code on my system, so there's more of a chance of me or someone else stumbling into a regression.

If I understand right, you want me to not bump at all, but I'm always bumping for the release, and given that, it's better to bump right away than wait until the release day.